src/test/mediapipe/graphadd.pbtxt

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
TO remove

Reply:
This calculator should be hidden and only for test purposes.

Reply:
Decided to leave this calculator for cross verification between using pure C-API and Adapter.